src/transforms/identity.jl

```diff
@@ -6,12 +6,6 @@
     Identity()
 
 The identity transform `x -> x`.
-
-# Examples
-
-```julia
-Identity()
-```
```

Comment:
Is this intended? The PR is about removing exports right?

Reply:
It's such a small change that I thought it could be included in this PR.
Want me to do a PR for this change?

Reply:
No problem, just wanted to double check it was an intentional change. 👍🏽